apps/events-helsinki/src/domain/event/EventPageContainer.tsx

```diff
@@ -10,7 +10,7 @@ import { useRouter } from 'next/router';
 import React from 'react';
 import { Link } from 'react-helsinki-headless-cms';
 
-import { ROUTES } from '../../constants';
+import { MAIN_CONTENT_ID, ROUTES } from '../../constants';
```

Comment:
FYI: when we initialized the monorepo, there was a linter that wined about this all caps variables. Anyway, I find it better like this